src/Server/Main.idr

```diff
@@ -98,7 +98,7 @@ runServer = do
   catch (processMessage method msg)
         (\err => do
           logString Error (show err)
-          resetContext)
+          resetContext "")
```

Comment:
I think the convention is "(interactive)", but these aren't well documented, so I could be wrong. There are other places like this elsewhere, I'll just stick "(interactive)" there.